src/assets/scss/checkbox.scss

```diff
@@ -1,20 +1,26 @@
-fieldset {
-    border: none;
-    display: flex;
-    flex-direction: column;
-    margin: 20px;
-    align-items: flex-start;
-    min-width: 400px;
+.container-checkbox-component{
```

Comment:
That's a really big class name to deal with simple styles, maybe you should do something like:
class="checkbox-container checkbox-group checkbox-filled"...
Like this you can optimize your SCSS to organize it, check this out:
.checkbox- {
....&container {
........// styles for container
....}

....&group {
........// styles for group
....}

....&filled {
........// styles for filled
....}
}

Reply:
I decrease the name, but the way you want to do I do not understand, so modify to your liking if you want to start from scratch feel free, I understood how fabio did, but as this component is just a just do not know how to apply this.